src/features/OnHover.ts

```diff
@@ -0,0 +1,31 @@
+import { Subject } from 'rxjs';
+import { bufferTime } from 'rxjs/operators';
+import { FeatureBase } from './FeatureBase';
+
+export class OnHover extends FeatureBase {
+    private hoverButtonSource = new Subject()
+
+    initializeOn(grid: any) {
+        this.hoverButtonSource
+            .pipe(bufferTime(100))
```

Comment:
buffer time works perfectly under this use case.
reference: https://www.learnrxjs.io/learn-rxjs/operators/transformation/buffertime